Uplift/Controllers/FilterViewController.swift

```diff
@@ -149,7 +149,7 @@ class FilterViewController: UIViewController, RangeSeekSliderDelegate {
         classTypeDropdown.bounces = false
 
         classTypeDropdown.register(DropdownViewCell.self, forCellReuseIdentifier: DropdownViewCell.identifier)
-        classTypeDropdown.register(DropdownHeaderView.self, forHeaderFooterViewReuseIdentifier: DropdownHeaderView.identifier)
+        classTypeDropdown.register(OldDropdownHeaderView.self, forHeaderFooterViewReuseIdentifier: OldDropdownHeaderView.identifier)
```

Comment:
why did we rename this to `OldDropdownHeaderView`?

Reply:
The old implementation of a dropdown view also had a class named `DropdownHeaderView`, which caused name conflicts with my `DropdownHeaderView`. I intend to refactor the old dropdown views later on to use the new dropdown view, but for now I just renamed the old header class to something else to fix the name conflict.